blog/_single.html

```diff
@@ -38,7 +38,7 @@ <h1 class="header-slug">
                 Stay informed
             </span>
         </h1>
-        <div class="block block__sub u-mt0">
+        <div class="block block__sub block__flush-top">
```

Comment:
What's the rule on stacking classes, more than three? Should we create one if we don't have one because I never know if I should comment on them.